.devops/deploy-pipelines.yml

```diff
@@ -112,7 +112,7 @@ stages:
               azureSubscription: '$(STAGING_AZURE_SUBSCRIPTION)'
               destination: 'AzureBlob'
               storage: '$(STAGING_STORAGE_ACCOUNT_NAME)'
-              containerName: '$(BLOB_CONTAINER_NAME)'
+              containerName: 'xxx'
```

Comment:
wat?

Reply:
I'm just trying to debug if I have a problem of naming or permission on the $web container :-)